tsconfig.json

```diff
@@ -1,6 +1,39 @@
 {
-    "extends": "./configs/base.tsconfig.json",
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "compilerOptions": {
+        "target": "ES2017",
+        "module": "commonjs",
+        "lib": [
+            "ES2017",
+            "DOM"
+        ],
+        "types": [
+            "reflect-metadata"
+        ],
+        "sourceMap": true,
+        "declaration": true,
+        "declarationMap": true,
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "noImplicitReturns": true,
+        "noImplicitOverride": true,
+        "noUnusedLocals": true,
+        "strictNullChecks": true,
+        "experimentalDecorators": true,
+        "emitDecoratorMetadata": true,
+        "esModuleInterop": true,
+        "jsx": "react"
+    },
     "include": [
-        "packages/*/src/**/*.ts",
+        "**/src/**/*.ts",
+        "**/src/**/*.tsx",
+        "**/packages/*/*/*.ts",
+        "**/packages/*/*/*.js",
+        "*.js",
+        "*.ts"
```

Comment:
Why so redundant?

Reply:
It is just for configuration files (vite and eslint).
It is actually wrong I just realized. It should be:
```
"**/packages/*/*.ts",
"**/packages/*/*.js"
```

Reply:
What files are you including with those two entries?

Reply:
Any `vite.config.ts` or `.eslintrc.js`. Lint is checked there as well as well as VSCode is looking at them. All packages have their own include definition.

Reply:
I don't think it's necessary to lint the configs.

Reply:
Ok, remove below. I will merge once the build pipeline was successful.